service/dap/server_test.go

```diff
@@ -345,135 +345,163 @@ func TestSetBreakpoint(t *testing.T) {
 	})
 }
 
+var expectFrames = func(t *testing.T, stResp *dap.StackTraceResponse, wantStart int, wantFrames int, wantTotal int, wantLine int) {
```

Comment:
```
func expectFrames(t *testing.T, ...)
```

And please add a comment for this function

Reply:
Done.